ppmpy/ppm.py

```diff
@@ -904,22 +904,33 @@ def compute_N2(self, fname, num_type='ndump', radeos=True):
 
         return N2
 
-    def compute_m(self, fname, num_type='ndump'):
+    def compute_m(self, fname, num_type='ndump', minner=0):
         if self.__isyprofile:
             r = self.get00('Y', fname, num_type=num_type, resolution='l')
             rho = self.get('Rho', fname, num_type=num_type, resolution='l')
+            rinner = 0
+            minner = 0
```

Comment:
add warning if minner in input is not 0 and isyprofile; there is a warning and error framework, use that